lighthouse-core/audits/byte-efficiency/offscreen-images.js

```diff
@@ -72,7 +72,8 @@ class OffscreenImages extends Audit {
     const url = URL.getDisplayName(image.src, {preserveQuery: true});
     const totalPixels = image.clientWidth * image.clientHeight;
     const visiblePixels = this.computeVisiblePixels(image.clientRect, viewportDimensions);
-    const wastedRatio = 1 - visiblePixels / totalPixels;
+    // Treat images with 0 area as if they're offscreen
```

Comment:
> See any ways around such a false positive?

there was @paulirish's suggestion to look at what images were actually decoded, indicating that Chrome thought that the image was needed. That makes other aspects of the audit more difficult to do.